hms-app/src/pages/IdeaPortal.tsx

```diff
@@ -202,35 +76,33 @@ function IdeaPortal() {
         />
       </Group>
 
-      {isHackathonError && (
-        <div>
-          <h3>Error loading hackathons</h3>
-          <p>something went wrong.</p>
-        </div>
-      )}
-      {isHackathonLoading && (
-        <div>
-          <h3>Hackathon details are loading...</h3>
-        </div>
-      )}
+      <RelevantIdeasLoader
```

Comment:
I still don't know about calling it a `Loader` either it is a visual component is it not?